opentelemetry-exporter-cloud-monitoring/tests/test_cloud_monitoring.py

```diff
@@ -70,16 +70,18 @@ def __init__(
 
 
 class TestCloudMonitoringMetricsExporter(unittest.TestCase):
-    def setUp(self):
-        self.client_patcher = mock.patch(
+    @classmethod
+    def setUpClass(cls):
+        cls.client_patcher = mock.patch(
```

Comment:
This patch might be better off in `setUp()` or iirc you can decorate the class with it. It doesn't look like you're using the mock in any of the test cases, but it would accumulate state (e.g. `mock.method_calls`) across test cases.

My original comment was regarding immutable constants